dbms/src/Storages/KVStore/MultiRaft/ApplySnapshot.cpp

```diff
@@ -196,9 +233,12 @@ void KVStore::onSnapshot(
                             new_key_range.toDebugString(),
                             keyspace_id,
                             table_id);
-                        dm_storage->deleteRange(old_key_range, context.getSettingsRef());
-                        // We must flush the deletion to the disk here, because we only flush new range when persisting this region later.
-                        dm_storage->flushCache(context, old_key_range, /*try_until_succeed*/ true);
```

Comment:
I think we should directly remove all these lines. Leaving the logging of clear old range but actually not remove it is really misleading.
https://github.com/pingcap/tiflash/blob/14007a151c5a838e9faaa5bde4558d7d0fe80213/dbms/src/Storages/KVStore/MultiRaft/ApplySnapshot.cpp#L218-L235

Reply:
We can change the log. However, we have to log this because it shows the region range change while applying snapshot